geoxarray/crs/_proj4.py

```diff
@@ -22,28 +22,29 @@
 
 """
 
```

Comment:
Is there a need to keep this file around?

Reply:
You tell me. How directly does cartopy support pyproj CRS objects? I suppose this is the latest: https://pyproj4.github.io/pyproj/stable/crs_compatibility.html#preparing-pyproj-crs-crs-for-cartopy-crs-crs ?

Reply:
The methods in there just use the `pyproj.CRS` object and I was wondering if it was something that used to be more complex and was simplified later. I was thinking of using the `pyproj.CRS` object directly. Just something to think about. Feel free to ignore if it is irrelevant.

Side note: Even this doesn't do a great job preserving the CRS information: https://pyproj4.github.io/pyproj/stable/crs_compatibility.html#preparing-pyproj-crs-crs-for-cartopy-crs-crs . It will be nice when authority codes/WKT are supported by cartopy.

Reply:
:facepalm: Obviously I assumed this was the `_cartopy.py` module that is also in this package and was copied from pyresample. The `proj4_str_to_dict` is used in that module and is likely just a quick swap-out for something in cartopy or in pyresample. You're right that pyproj easily handles this now.

Regarding cartopy and CRS: I rarely use authority codes as most of the satellite data I work with is either ungridded data or isn't a registered CRS with any authority (ex. GOES-WEST/EAST ABI full disk). So I'd still need something for arbitrary projection information.